snuba_sdk/mql/mql.py

```diff
@@ -109,6 +113,10 @@ class InvalidMQLQueryError(Exception):
     "/": ArithmeticOperator.DIVIDE.value,
 }
 
+UNARY_OPERATORS: Mapping[str, str] = {
+    "-": ArithmeticOperator.MINUS.value,
```

Comment:
Clickhouse uses the `negate` function for this, Clickhouse expects the `minus` function to have 2 arguments.